.gitignore

```diff
@@ -106,3 +106,7 @@
 /tests/check/test-lib-introspection.sh.trs
 /tests/kola
 _libs
+
+# for clangd: https://clangd.llvm.org/installation#project-setup
+/compile_commands.json
```

Comment:
I'm not opposed to to this but I also think it's probably better in your global per-user `~/.gitignore`.  https://sebastiandedeyne.com/setting-up-a-global-gitignore-file/

Reply:
Oh neat! I didn't realize that was a thing. Happy to revert that commit.